configuration/external_parameters.rst

```diff
@@ -80,24 +40,86 @@ You can now reference these parameters wherever you need them.
 
         <doctrine:config>
             <doctrine:dbal
-                driver="pdo_mysql"
-                dbname="symfony_project"
-                user="%database.user%"
-                password="%database.password%"
+                host="%env(DATABASE_HOST)%"
             />
         </doctrine:config>
 
     .. code-block:: php
 
         $container->loadFromExtension('doctrine', array(
             'dbal' => array(
-                'driver'   => 'pdo_mysql',
-                'dbname'   => 'symfony_project',
-                'user'     => '%database.user%',
-                'password' => '%database.password%',
+                'host' => '%env(DATABASE_HOST)%',
             )
         ));
 
+You can also give the ``env(...)`` parameters a default value: the default value
+will be used whenever the corresponding environment variable is *not* found:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        #app/config/parameters.yml
+
+        parameters:
+            env(DATABASE_HOST): localhost
```

Comment:
No problem @mickadoo thanks for test-casing this documentation proposal.

I think we should include the  `database_driver: '%env(DATABASE_DRIVER)%'` line in this example to avoid confusing for other readers.